frontend/src/bundles/auth/auth-api.ts

```diff
@@ -44,6 +44,14 @@ class AuthApi extends BaseHttpApi {
 
         return await response.json<AuthResponseDto>();
     }
+
+    public async logout(): Promise<void> {
+        await this.load(this.getFullEndpoint(AuthApiPath.LOGOUT, {}), {
+            method: 'GET',
+            hasAuth: true,
+            contentType: ContentType.JSON,
+        });
+    }
```

Comment:
why do we need this request?

Reply:
to set token to null in db, or it is not necessary?

Reply:
but we don't have token in the database currently

Reply:
Didn’t notice it 🤦‍♂️